test/nn/test_pooling.py

```diff
@@ -158,14 +147,6 @@ def test_adaptive_pooling_size_none(self):
                 output = module(input)
                 self.assertEqual(output.size(), (4,) + (2,) * (numel - 1) + (4,))
 
-    @unittest.skipIf(TEST_WITH_UBSAN, "signed integer overflow error with UBSAN")
```

Comment:
Should we skip the error inputs test when people are testing with a sanitizer?

Reply:
Thanks for the question, @mruberry. But just wondering why you think we need to ship this case?

Both raise errors for different reasons. Maybe adding a comment makes more sense. WDYT?
- **With UBSAN**: `integer multiplication overflow`
- **Without UBSAN**: `Storage size calculation overflowed`

Reply:
When PyTorch tests with its undefined behavior sanitizer, will `assertRaisesRegex` catch errors arising from the sanitizer?

However, I still think we should skip `test_errors` when running with a sanitizer. The test should be testing invalid inputs, after all. It seems like a lot of additional complexity to deal with a sanitizer when we're already on an invalid path that throws an error.

Reply:
@mruberry, I've updated the PR. Please take another look. Thanks!